app/src/androidTest/java/ch/epfl/sdp/blindly/recording/RecordingActivityTest.kt

```diff
@@ -34,15 +41,40 @@ private const val TEST_MAXIMUM_AUDIO_DURATION = 13000
 private const val FIVE_SECONDS = 5000L
 private const val TWO_SECONDS = 2000L
 
-
-@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
 class RecordingActivityTest {
-    private val initBundle = Bundle()
-    val intent = Intent(ApplicationProvider.getApplicationContext(), RecordingActivity::class.java)
-        .putExtra(AUDIO_DURATION_KEY, TEST_MAXIMUM_AUDIO_DURATION)
+    private val TEST_USER = User.Builder()
+        .setUsername(CORRECT_NAME)
+        .setBirthday(TEST_BIRTHDAY)
+        .setGender(TEST_GENDER_WOMEN)
+        .setSexualOrientations(TEST_SEXUAL_ORIENTATIONS)
+        .setPassions(TEST_PASSIONS)
+    private val SERIALIZED = Json.encodeToString(TEST_USER)
+    private lateinit var scenario: ActivityScenario<RecordingActivity>
```

Comment:
It's a detail, but maybe for consistency (with the other tests)  it would be better to keep the activityRule = activityScenarioRule
And then call activityRule.scenario to get the scenario, rather than have a private lateinit var

Reply:
I changed it :)